pipelines/performance-tests-pipeline.yml

```diff
@@ -940,3 +994,106 @@ jobs:
     input_mapping: {performance-tests-repo: performance-tests-repo}
   on_failure: *slack_performance_tests_failure
   on_success: *slack_performance_tests_finished
+
+- name: "Trigger Terraform"
+  serial: true
+  serial_groups: [
+    scale-down,
+    scale-apps,
+    performance-tests,
+    action-scheduler,
+    action-worker,
+    case-api,
+    case-processor,
+    fieldwork-adapter,
+    notify-processor,
+    uac-qid-service,
+    pubsubsvc,
+    print-file-service,
+    exception-manager,
+    toolbox,
+    event-latency-monitor,
+    rabbitmonitor]
+  plan:
+    - get: every-minute
+
+# Run Terraform
+- name: "Run Terraform"
+  disable_manual_trigger: true
+  serial: true
+  serial_groups: [scale-down,
+                  scale-apps,
+                  performance-tests,
+                  action-scheduler,
+                  action-worker,
+                  case-api,
+                  case-processor,
+                  fieldwork-adapter,
+                  notify-processor,
+                  uac-qid-service,
+                  pubsubsvc,
+                  print-file-service,
+                  exception-manager,
+                  toolbox,
+                  event-latency-monitor,
+                  rabbitmonitor]
+  plan:
+    - get: every-minute
+      trigger: true
+      passed: ["Trigger Terraform"]
+    - get: census-rm-terraform
+    - get: census-rm-deploy
+    - task: "Run Terraform"
+      file: census-rm-deploy/tasks/terraform-env.yml
+      params:
+        ADMIN_SERVICE_ACCOUNT_JSON: ((gcp.service_account_json))
+        ENV: performance
+        VAR_FILE: ./tfvars/census-rm-performance.tfvars
+        KUBERNETES_CLUSTER: rm-k8s-cluster
+      input_mapping: {census-rm-terraform: census-rm-terraform}
+
+# Run Rabbit Helm
+- name: "Run Rabbit Helm"
+  disable_manual_trigger: true
+  serial: true
+  serial_groups: [scale-down,
+                  scale-apps,
+                  performance-tests,
+                  action-scheduler,
+                  action-worker,
+                  case-api,
+                  case-processor,
+                  fieldwork-adapter,
+                  notify-processor,
+                  uac-qid-service,
+                  pubsubsvc,
+                  print-file-service,
+                  exception-manager,
+                  toolbox,
+                  event-latency-monitor,
+                  rabbitmonitor]
+  plan:
+    - get: every-minute
+      trigger: true
+      passed: ["Run Terraform"]
```

Comment:
Do we want this to be dependant on the terraform job? They are configured independently.

Reply:
I was thinking that it would run like a manually triggered pipeline which resets the env to a known good state

Reply:
So whenever there's any infrastructure change we just kick the whole section off? Makes sense 👍